content/applications/marketing/email_marketing.rst

```diff
@@ -0,0 +1,10 @@
+:nosearch:
```

Comment:
When a page has children pages, it isn't visible by default. If you add the "essentials" in this page, you'd need these options:
```suggestion
:show-content:
:show-toc:
```